seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -374,7 +374,7 @@
       "translations": {
         "de": {
           "description": [
-            "Kommentieren erlaubt dir Kommentare innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
+            "Kommentieren erlaubt es dir, Bemerkungen innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
             "Dies ist auch ein guter Weg, um Code inaktiv zu machen ohne diesen komplett löschen zu müssen.",
             "Du kannst einen Kommentar mit <code>&#60;!--</code> starten und ihn mit <code>--&#62;</code> wieder beenden.",
```

Comment:
change starten to beginnen

since we used that terminology in the next section. Should be more consistent.